management/src/main/java/org/ehcache/management/providers/statistics/EhcacheStatisticsProvider.java

```diff
@@ -55,6 +69,17 @@ protected void dispose(ExposedObject<CacheBinding> exposedObject) {
   }
 
   @Override
+  public final Collection<? extends Descriptor> getDescriptors() {
+    Collection<StatisticDescriptor> capabilities = new HashSet<StatisticDescriptor>();
+    for (ExposedObject o : getExposedObjects()) {
+      capabilities.addAll(((StandardEhcacheStatistics) o).getDescriptors());
+    }
+    List<StatisticDescriptor> list = new ArrayList<StatisticDescriptor>(capabilities);
+    Collections.sort(list, STATISTIC_DESCRIPTOR_COMPARATOR);
```

Comment:
Yes, should have been! Now... Is it worth loosing another  1h for the new build to pass for this small thing and delaying more the PR for M&M ? :-S